config/keys.go

```diff
@@ -61,7 +61,7 @@ func Endpoint() string {
 
 func Title() string {
 	dir, _ := os.Getwd()
-	return TryStrings(StringFlag(flags.Title), file.Title(), filepath.Base(dir))
+	return TryStrings(StringFlag(flags.Title), file.Title(), Project(), filepath.Base(dir))
```

Comment:
Just so I'm on the same page as you here. Did you do this because you noticed that if `file.CLI.Project` was not set then we would fallback to `filepath.Base(dir)`?  This way seems to ensure that `Title` will always be at least in sync with `Project`.

Reply:
This shouldn't change any behavior because of the update I made to `func (file File) Title()`, which will now simply return whatever is set in the file (even if it's empty); so here on this line we need to call `Project()` to be consistent with the way we establish the value of any setting.